kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -12,31 +12,44 @@ import { PageNames } from '../constants';
 import reportRoutes from './reportRoutes';
 import planRoutes from './planRoutes';
 
+function classIdParamRequiredGuard(toRoute, subtopicName) {
+  if (store.getters.userIsMultiFacilityAdmin && !toRoute.params.classId) {
```

Comment:
This says classIdParamRequired guard, but it only seems to be accounting for the situation where there is no classId but multiple facilities - what about when there is no classId but multiple classes?